bindings/python/iota_sdk/types/output.py

```diff
@@ -22,6 +31,37 @@ class OutputType(IntEnum):
 
 @dataclass
 class Output():
+    """Output.
+
+    **Attributes**
+    type : int
+        Output type
+    amount : str
+        Output amount as string
+    unlockConditions : List[UnlockCondition]
+        Unlock conditions
+    aliasId : HexStr, optional
+        Alias ID
+    nftId : HexStr, optional
+        NFT ID
+    stateIndex : int, optional
+        State index
+    stateMetadata : HexStr, optional
+        State metadata
+    foundryCounter : int, optional
+        Foundry counter
+    features : List[Feature], optional
+        Features
+    nativeTokens : List[NativeToken], optional
+        Native tokens
+    immutableFeatures : List[Feature], optional
+        Immutable features
+    serialNumber : int, optional 
+        Serial number
+    tokenScheme : TokenScheme, optional
```

Comment:
Remove the types?

Reply:
yeah ... that went into `part 2` (see previous commit message :stuck_out_tongue_closed_eyes: )